seahub/api2/endpoints/admin/groups.py

```diff
@@ -42,12 +42,24 @@ class AdminGroups(APIView):
     permission_classes = (IsAdminUser,)
 
     def get(self, request):
-        """ List all groups
+        """ List all groups / search group by name
 
         Permission checking:
         1. Admin user;
         """
 
+        # search groups by name
```

Comment:
http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#advanced-queries
GET /tickets?q=return&state=open&sort=-priority,created_at - Retrieve the highest priority open tickets mentioning the word 'return'